app.rb

```diff
@@ -0,0 +1,171 @@
+require_relative 'student_class'
+require_relative 'teacher_class'
+require_relative 'book'
+require_relative 'rental'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+  end
+
+  def colorize_output(color_code, statements)
+    puts "\e[#{color_code}m#{statements}\e[0m"
+  end
+
+  def colorize_outprint(color_code, statements)
+    print "\e[#{color_code}m#{statements}\e[0m"
+  end
+
+  def list_all_books
+    if @books.empty?
+      colorize_output(31, 'Please insert books first!!')
+    else
+      @books.each do |book|
+        puts "Title: #{book.title}, Author: #{book.author}"
+      end
+    end
+    display_list
+    continue
+  end
+
+  def list_all_people
+    if @people.empty?
+      colorize_output(31, 'Please insert people first!!')
+    else
+      @people.each do |person|
+        puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+      end
+    end
+    display_list
+    continue
+  end
+
+  def create_person
+    colorize_outprint(35, 'Do you want to create a student (1) or a teacher (2)? [Input the number]: ')
+
+    case gets.chomp
+    when '1'
+      colorize_output(36, 'Enter Student details')
+      colorize_outprint(35, 'Name: ')
+      name = gets.chomp
+
+      colorize_outprint(35, 'Age: ')
+      age = gets.chomp
+
+      colorize_outprint(35, 'Has parent permission? [Y/N]: ')
+      permission = gets.chomp.downcase
+      case permission
+      when 'y' then parent_permission = true
+      when 'n' then parent_permission = false
+      end
+
```

Comment:
- [ ] Creating a teacher works fine but there is still an error when creating a student on your app. Kindly check it out.

![image](https://user-images.githubusercontent.com/98453979/224391522-1f151277-49de-49ae-9619-01d28f1a348d.png)

![image](https://user-images.githubusercontent.com/98453979/224391581-054f90bf-3c4f-4324-a5ea-fb591abac33d.png)

Reply:
Thanks @frank1738 for reviewing this PR